app/services/google_translation_service.rb

```diff
@@ -62,4 +68,12 @@ def transform_translations_for_caching(listing_id, keys, translations)
 
     return_value
   end
+
+  def logger(message, error: false)
```

Comment:
Can we call this google_translation_error or something similar? So that it makes it clear when it's being called that we are calling a specific error handler for this service